contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol

```diff
@@ -72,9 +78,11 @@ struct LimitsListPacked {
     uint16 maxValidatorExitRequestsPerReport;
     uint64 requestTimestampMargin;
     uint64 maxPositiveTokenRebase;
+    uint64 maxExitedValidatorsRatePerDay;
+    uint64 maxAccountingExtraDataListItemsCount;
```

Comment:
can be smaller (the idea is to fit the structure into the single slot)

Reply:
if #589 is merged, the type of `maxAccountingExtraDataListItemsCount` can be set as low as `uint24`

Reply:
churn limit is 1800 per day (?), thus `maxExitedValidatorsRatePerDay` can be a `uint16` (needs checking)

Reply:
churn limit is 60 validators at max (roughly all Eth TVL is staked), i.e., 13500 per day
perfectly feats into `uint16`